internal/common/constant/deprecation.go

```diff
@@ -1,6 +1,7 @@
 package constant
 
 const (
+	DeprecationParam                           = "This parameter is deprecated."
```

Comment:
don't we want to give more info about removal date or version?

Reply:
From the PD deprecation message example there is no reference to date or target version, however it does also mention following the 12 months deprecation cycle of the old API. Would be good to confirm with @Zuhairahmed if we do in fact want to include a date in this case.

Reply:
at this point we don't want to provide a date